examples/web/src/profiles/UseBlockAndUnblockProfiles.tsx

```diff
@@ -11,6 +11,15 @@ import { ErrorMessage } from '../components/error/ErrorMessage';
 import { Loading } from '../components/loading/Loading';
 import { ProfileCard } from './components/ProfileCard';
 
+function SponsoredFlag() {
+  return (
+    <label>
+      <input type="checkbox" name="sponsored" value="on" defaultChecked={true} />
+      sponsored
+    </label>
+  );
+}
```

Comment:
I don't see the point in the level of indirection introduces by this component.

Reply:
fair, removed